pkg/pool-weighted/test/IndexedPool.test.ts

```diff
@@ -0,0 +1,135 @@
+import { ethers } from 'hardhat';
```

Comment:
File name should probably be IndexPool.test.ts

Reply:
Done